x-pack/plugins/apm/server/lib/correlations/get_correlations_for_failed_transactions/index.ts

```diff
@@ -134,26 +134,24 @@ export async function getErrorRateTimeSeries({
       extended_bounds: { min: start, max: end },
     },
     aggs: {
-      // TODO: add support for metrics
-      outcomes: getOutcomeAggregation({ searchAggregatedTransactions: false }),
+      outcomes: getOutcomeAggregation(),
     },
   };
 
   const perTermAggs = topSigTerms.reduce(
     (acc, term, index) => {
-      acc[`term_${index}`] = {
+      acc[`term_${String(index)}`] = {
```

Comment:
Wondering why this is now necessary

Reply:
It might not be. I was trying to use template literal types to make it work, but those don't really work the way I expected it to, at least not as index/lookup keys.